src/app/features/tech-record/tech-record.component.ts

```diff
@@ -14,11 +16,20 @@ import { Observable } from 'rxjs';
 export class TechRecordComponent {
```

Comment:
Should this be implementing OnChanges if you're using it ?

Reply:
Just realised the comment isn't showing much but on line 23 you've got the ngOnChanges hook and no implement, just for further clarification!